src/mpid/common/shm/mpidu_init_shm.h

```diff
@@ -10,8 +10,11 @@
 
 #include "mpidu_shm.h"
 
+/* One cache line per process should be enough for all cases */
+#define MPIDU_INIT_SHM_BLOCK_SIZE MPIDU_SHM_CACHE_LINE_LEN
+
 typedef struct MPIDU_Init_shm_block {
-    char block[MPIDU_SHM_CACHE_LINE_LEN * 16];
+    char block[MPIDU_INIT_SHM_BLOCK_SIZE];
 } MPIDU_Init_shm_block_t;
```

Comment:
@hzhou I renamed this and made it only one cache line long as currently we don't need more space. We can change it if needed